app/views/kudos/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="bg-light shadow-lg p-4 mb-5 rounded">
+<h2>Create New Kudo!</h2>
+
+<%= render 'form', kudo: @kudo %>
+
+<hr class="my-4">
+<%= link_to 'Back To Kudos List', kudos_path, class:"btn btn-secondary" %>
+</div>
```

Comment:
missing newline at the end of the file - you can configure your IDE to take care of it for you, when you are saving the file :)